pypeit/spectrographs/gemini_gmos.py

```diff
@@ -507,8 +507,8 @@ def bpm(self, filename, det, shape=None, msbias=None):
 
             # Apply the mask
             xbin = int(binning.split(' ')[0])
-            if xbin != 2:
-                embed()
+            #if xbin != 2:
+            #    embed()
```

Comment:
remove the comments here and on line 492?

Reply:
done